react/README.md

```diff
@@ -62,16 +63,34 @@
     const reservationItem = <ReservationCard />;
     ```
 
-    **Component Naming**: Convert snake_case filename to PascalCase component name. For example, `reservation_card.js` should have a reference name of `ReservationCard`. However, for root components of a directory, use `index.js` as the filename and use the directory name as the component name:
+    **Component Naming**: Convert snake_case filename to PascalCase component name. For example, `reservation_card.js` should have a reference name of `ReservationCard`.
     ```javascript
     // bad
-    const Footer = require('./Footer/footer.js')
+    const Footer = require('./footer_component.js');
 
+    // good
+    const FooterComponent = require('./footer.js');
```

Comment:
Made it `const FooterComponent = require('./footer_component.js');` since it's more PascalCasey than just Footer.